osgp/platform/osgp-core/src/main/resources/db/migration/V20201112082800000__Adds_LTE_M_and_GRPS_protocol_info.sql

```diff
@@ -0,0 +1,130 @@
+DO
+$$
+begin
+
+if not exists (
+    select 1
+    from   protocol_info
+    where  protocol = 'SMR_LTE_M'
+    and    protocol_version  = '5.1') then
+
+insert into protocol_info(
+    creation_time,
+    modification_time,
+    version,
+    protocol,
+    protocol_version,
+    outgoing_requests_property_prefix,
+    incoming_responses_property_prefix,
+    incoming_requests_property_prefix,
+    outgoing_responses_property_prefix,
+    parallel_requests_allowed)
+values (
+    '2020-11-12 00:00:00',
+    '2020-11-12 00:00:00',
+    0,
+    'SMR_LTE_M',
+    '5.1',
+    'jms.protocol.dlms.outgoing.lte_m.requests',
+    'jms.protocol.dlms.incoming.responses',
+    'jms.protocol.dlms.incoming.requests',
+    'jms.protocol.dlms.outgoing.responses',
+    true);
+
+end if;
+
+if not exists (
+    select 1
+    from   protocol_info
+    where  protocol = 'SMR_LTE_M'
+    and    protocol_version  = '5.0.0') then
+
+insert into protocol_info(
+    creation_time,
+    modification_time,
+    version,
+    protocol,
+    protocol_version,
+    outgoing_requests_property_prefix,
+    incoming_responses_property_prefix,
+    incoming_requests_property_prefix,
+    outgoing_responses_property_prefix,
+    parallel_requests_allowed)
+values (
+    '2020-11-12 00:00:00',
+    '2020-11-12 00:00:00',
+    0,
+    'SMR_LTE_M',
+    '5.0.0',
+    'jms.protocol.dlms.outgoing.lte_m.requests',
+    'jms.protocol.dlms.incoming.responses',
+    'jms.protocol.dlms.incoming.requests',
+    'jms.protocol.dlms.outgoing.responses',
+    true);
+
+end if;
+
+if not exists (
```

Comment:
volgens mij hoef je deze niet toe te voegen, want dat is de default (GPRS) , ik heb toen voor CDMA alleen de CDMA toegevoegd. Als je deze toe wilt voegen moet je eigenlijk de bestaande verwijderen.

Reply:
Ik heb geen 'default' meer. We hebben aparte protocol adapters voor CDMA, GRPS en LTE-M.